crates/matrix-sdk-crypto/src/session_manager/group_sessions.rs

```diff
@@ -797,8 +797,19 @@ impl GroupSessionManager {
         let devices: Vec<_> = devices
             .into_iter()
             .flat_map(|(_, d)| {
-                d.into_iter()
-                    .filter(|d| matches!(outbound.is_shared_with(d), ShareState::NotShared))
+                d.into_iter().filter(|d| match outbound.is_shared_with(d) {
+                    ShareState::NotShared => true,
+                    ShareState::Shared(_msg_index, olm_wedging_index) => {
+                        // If the recipient device's Olm wedging index is
```

Comment:
This seems to be wrapped at a really small width, we wrap at 80 chars, `cargo fmt` will do this automatically for you if the lines are too long.